--- caja-dropbox-1.10.0-original/configure.ac	2015-07-18 10:34:43.611680792 -0400
+++ caja-dropbox-1.10.0/configure.ac	2015-07-18 10:44:09.898641007 -0400
@@ -32,7 +32,7 @@ PKG_CHECK_MODULES(GLIB, glib-2.0 >= $GLI
 
 AC_PATH_PROG([PYTHON], [python])
 
-AC_PATH_PROG([RST2MAN], [rst2man], [python rst2man.py])
+AC_PATH_PROG([RST2MAN], [rst2man.py], [python rst2man.py])
 AC_SUBST(RST2MAN)
 
 # define module checking macro
