--- mate-panel-1.10.1-orig/applets/wncklet/workspace-switcher.c	2015-07-18 13:32:27.604931580 -0400
+++ mate-panel-1.10.1/applets/wncklet/workspace-switcher.c	2015-07-18 13:35:52.766917166 -0400
@@ -146,6 +146,8 @@ static void window_manager_changed(WnckS
 		pager->wm = PAGER_WM_UNKNOWN;
 	else if (strcmp(wm_name, "Metacity (Marco)") == 0)
 		pager->wm = PAGER_WM_MARCO;
+	else if (strcmp(wm_name, "Marco") == 0)
+ 		pager->wm = PAGER_WM_MARCO;
 	else if (strcmp(wm_name, "Compiz") == 0)
 		pager->wm = PAGER_WM_COMPIZ;
 	else
