--- mate-settings-daemon-1.10.1-original/plugins/mouse/msd-mouse-manager.c	2015-07-18 12:01:51.893313473 -0400
+++ mate-settings-daemon-1.10.1/plugins/mouse/msd-mouse-manager.c	2015-07-18 12:07:58.461287719 -0400
@@ -576,7 +576,7 @@ set_disable_w_typing (MsdMouseManager *m
 
         if (state && touchpad_is_present ()) {
                 GError *error = NULL;
-                char *args[5];
+                char *args[6];
 
                 if (manager->priv->syndaemon_spawned)
                         return 0;
@@ -585,7 +585,8 @@ set_disable_w_typing (MsdMouseManager *m
                 args[1] = "-i";
                 args[2] = "0.5";
                 args[3] = "-k";
-                args[4] = NULL;
+                args[4] = "-R";
+                args[5] = NULL;
 
                 if (!g_find_program_in_path (args[0]))
                         return 0;
